workspaces/ui-v2/src/optic-components/pages/diffs/CapturePage.tsx

```diff
@@ -191,7 +189,12 @@ function DiffCaptureResults() {
                 disableRipple
                 button
                 key={index}
-                onClick={handleChangeToEndpointPage(i.pathId, i.method)}
+                onClick={() =>
+                  !done && handleChangeToEndpointPage(i.pathId, i.method)
+                }
+                style={{
+                  cursor: done ? 'default' : 'pointer',
+                }}
```

Comment:
Since we autodirect back to the capture page, i updated this to not allow navigation to the review page - otherwise you navigate and you get navigated back immediately

<img width="988" alt="Screen Shot 2021-05-03 at 5 18 25 PM" src="https://user-images.githubusercontent.com/18374483/116948217-b088d880-ac33-11eb-99b3-c8345a1ac48c.png">